flocker/node/agents/_logging.py

```diff
@@ -0,0 +1,66 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+from eliot import Field, ActionType, MessageType
+
+# Begin: Helper datastructures to log all
+# IBlockDeviceAPI calls from AWS storage driver using Eliot.
+
+# An OPERATION is a list of:
+# IBlockDeviceAPI name, positional arguments, keyword arguments.
+OPERATION = Field.forTypes(
+    u"operation", [list],
```

Comment:
> Is list a valid Field type?

Yes, it is: https://github.com/ClusterHQ/eliot/blob/master/eliot/_validation.py#L124